pages/_app.js

```diff
@@ -39,7 +41,10 @@ export default function App({ Component, pageProps }) {
         }}
       >
         <GlobalStyle />
-        <Component {...pageProps} projects={projects} />
+        <Layout></Layout>
+        <Wrapper>
+          <Component {...pageProps} projects={projects} />
+        </Wrapper>
```

Comment:
```suggestion
        <Layout>
          <Component {...pageProps} projects={projects} />
        </Layout>
```
Because we're using `Wrapper` inside of `Layout` we don't need to use the `Wrapper` by itself here.